src/glass/src/app/pages/welcome-page/welcome-page.component.html

```diff
@@ -0,0 +1,81 @@
+<div class="glass-welcome-page"
+     fxLayout="column"
+     fxLayoutAlign="center center">
+  <div class="card">
+    <div class="card-header">
+      <div class="card-title"
+           *ngIf="!checked"
+           translate>
+        Welcome to Aquarium
+      </div>
+      <div class="card-title"
+           *ngIf="checked"
+           translate>
+        Requirements report
+      </div>
+    </div>
+    <div class="card-body">
+      <div class="card-text"
+           *ngIf="!checked"
+           translate>
+        In order to get this node started and ensure anything works as smoothly as possible, we need to check if it
+        fulfills at least a minimum set of requirements.
+      </div>
+      <div *ngIf="checked">
+        <div *ngIf="qualified; then qualifiedTpl else notQualifiedTpl"></div>
+      </div>
+    </div>
+    <div class="card-footer">
+      <div class="card-actions">
+        <button type="button"
+                class="btn btn-warning"
+                (click)="checkRequirements()"
+                *ngIf="!checked">
+          <span translate>Check requirements now</span>
+        </button>
+        <button type="button"
+                class="btn btn-warning"
+                routerLink="/installer/install-mode"
+                *ngIf="checked && qualified">
+          <span translate>Next</span>
+        </button>
+      </div>
+    </div>
+  </div>
+</div>
+
+<ng-template #qualifiedColTpl
+             let-value="value">
+  <i *ngIf="value"
+     class="text-success"
+     [class]="icons.check">
+  </i>
+  <i *ngIf="!value"
+     class="text-danger"
+     [class]="icons.minus">
+  </i>
+</ng-template>
+
+<ng-template #qualifiedTpl>
+  <glass-alert-panel type="success">
+    <span translate>Congratulation! The node fulfills the requirements. You can now proceed with the installation.</span>
```

Comment:
```suggestion
   Congratulations! You can now proceed.

Reply:
Keep these messages really short and simple. Debatably, you should remove the "congratulations!" but I like it, personally.